efficientdet/inference.py

```diff
@@ -477,11 +495,13 @@ def inference(self,
           self.model_name, images, **self.params)
       restore_ckpt(sess, self.ckpt_path, enable_ema=True, export_ckpt=None)
       # for postprocessing.
-      params.update(dict(batch_size=len(raw_images), disable_pyfun=False))
+      params.update(dict(batch_size=len(raw_images), disable_pyfun=True))
```

Comment:
Could you first set it to False. I will first merge your PR, and verify that disable_pyfun=True/False have the same outputs. After that, I will change the default value to True.

Reply:
Another choice is to have a class member as disable_pyfun and users can choose to set different values.